data/core/units/humans/Horse_Paladin.cfg

```diff
@@ -6,7 +6,7 @@
     image="units/human-loyalists/paladin/paladin.png"
     image_icon="units/human-loyalists/paladin/paladin.png~CROP(8,10,72,72)"
     profile="portraits/humans/paladin.webp"
-    hitpoints=65
+    hitpoints=6775
```

Comment:
@Hejnewar Bruh...XD..I mean what?

Reply:
Fair and Balanced as all things should be.